docs/docs/behaviors/layers.md

```diff
@@ -149,6 +149,72 @@ Example:
 
 It is possible to use "toggle layer" to have keys that raise and lower the layers as well.
 
+## Momentary Layer Lock
+
+Even if you mostly use [momentary layers](#momentary-layer) instead of `&to` or `&tog`, it's occasionally useful to permanently enable a layer without needing to hold anything down. Instead of creating an additional `&tog` or `&to` binding for each such layer, you can use `&molock`.
+
+If `&molock` is pressed while any number of `&mo` bindings are being held, those momentary layers will not be deactivated when the corresponding `&mo` key is released. As a result, those momentary layers become "locked" until that `&mo` key is pressed and released a second time or the layer becomes deactivated by some other means (e.g. a `&tog` binding for that layer or a `&to` binding for a lower one).
+
+If `&molock` is pressed while no `&mo` bindings are being held, it triggers a user-configurable fallback behavior. The default fallback behavior returns to the base layer (`&to 0`), deactivating any locked momentary layers in the process.
+
+### Behavior Binding
+
+- Reference: `&molock`
+
+Example:
+
+```dts
+&molock
+```
+
+Lock a symbol layer:
+
+```dts
+#define BASE 0
+#define SYMS 1
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        base_layer {
+            bindings = <&mo SYMS  &kp Z     &kp M      &kp K  >;
+        };
+        symbol_layer {
+            bindings = <&trans    &kp PLUS  &kp MINUS  &molock>;
+        };
+    };
+};
+```
+
+:::info
```

Comment:
I wouldn't put this in a note block, it should be good to have it in a single plain paragraph.